packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -541,10 +541,20 @@ export class StripeWebhookHandler extends StripeHandler {
         this.stripeHelper.getCustomerPaypalAgreement(customer);
       // The customer needs to be updated for their name.
       if (billingAgreementId) {
-        await this.paypalHelper.updateStripeNameFromBA(
-          customer,
-          billingAgreementId
-        );
+        try {
+          await this.paypalHelper.updateStripeNameFromBA(
+            customer,
+            billingAgreementId
+          );
+        } catch (err) {
```

Comment:
There should be a check to make sure its the error we expect, as this catches any error.